index.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+module.exports = {
+  extends: 'eslint:recommended',
+  env: { es6: true },
+  parserOptions: { ecmaVersion: 2015 },
```

Comment:
Why this specific version and not something newer?

Reply:
It reflects Node.js v6 which we currently support in `serverless` project. Anything greater will allow syntax that may crash v6.

We may also put greater version here, but then in `serverless` we should override it.

Reply:
Ok, that sounds reasonable. I was just asking because I'm not sure which projects use which Node.js version and maybe there's a tradeoff to be made here that we use newer version here and use v6 stuff only for the Framework (if that's the only project holding us back).

Hopefully we can get rid of Node.js 6 support soon.